cts-java/src/test/java/org/ga4gh/cts/api/Utils.java

```diff
@@ -125,6 +128,7 @@ public static String getValidReferenceId(Client client) throws AvroRemoteExcepti
         final SearchReferencesRequest refsReq = SearchReferencesRequest
                 .newBuilder()
                 .setReferenceSetId(refSets.get(0).getId())
+                .setMd5checksums(aSingle(TestData.REFERENCE_BRCA1_MD5_CHECKSUM))
                 .build();
         final SearchReferencesResponse refsResp = client.references.searchReferences(refsReq);
         assertThat(refsResp).isNotNull();
```

Comment:
Would also be nice to assert that you only receive exactly one reference from this query.